include/mgos_pppos.h

```diff
@@ -120,9 +120,13 @@ struct mgos_pppos_cmd {
  */
 bool mgos_pppos_run_cmds(int if_instance, const struct mgos_pppos_cmd *cmds);
 
-struct mg_str mgos_pppos_get_imei(int if_instance);
```

Comment:
don't change existing API. if you need FFI-able helpers, add them. they actually don't need to be public (i.e. exported in the header). add `_js` suffix at the end - i.e. `mgos_pppos_get_imei_js`, etc.

Reply:
Got!